modules/nw-metallb-configure-return-traffic-proc.adoc

```diff
@@ -0,0 +1,179 @@
+// Module included in the following assemblies:
+//
+// * networking/ovn_kubernetes_network_provider/configuring-egress-traffic-for-vrf-loadbalancer-services.adoc
+
+:_content-type: PROCEDURE
+[id="nw-metallb-configure-return-traffic-proc_{context}"]
+= Configuring symmetric routing using VRF-based networks with MetalLB
+
+You can configure symmetric network routing for applications behind a MetalLB service that require the same ingress and egress network paths. 
+
+This example associates a VRF routing table with MetalLB and an egress service to enable symmetric routing for ingress and egress traffic for pods behind a `LoadBalancer` service.
+
+.Prerequisites
+
+* Install the OpenShift CLI (`oc`).
+* Log in as a user with `cluster-admin` privileges.
+
+.Procedure
+
+. Create a `NodeNetworkConfigurationPolicy` custom resource (CR) to define the VRF instance.
+
+.. Create a file, such as `node-network-vrf.yaml`, with content like the following example:
++
+[source,yaml]
+----
+apiVersion: nmstate.io/v1
+kind: NodeNetworkConfigurationPolicy
+metadata:
+  name: vrfpolicy <1>
+spec:
+  nodeSelector: 
+    vrf: "true" <2>
+  maxUnavailable: 3 
+  desiredState:
+    interfaces:
+      - name: ens4vrf <3>
+        type: vrf <4>
+        state: up
+        vrf:
+          port:
+            - ens4 <5>
+          route-table-id: 2 <6>
+    routes: <7>
+      config:
+        - destination: 0.0.0.0/0 
+          metric: 150
+          next-hop-address: 192.168.130.1
+          next-hop-interface: ens4
+          table-id: 2
+    route-rules: <8>
+      config:
+        - ip-to: 172.30.0.0/16
+          priority: 998
+          route-table: 254
+        - ip-to: 10.132.0.0/14
+          priority: 998
+          route-table: 254
+----
+<1> Name of the policy.
+<2> This example applies the policy to all nodes with the label `vrf:true`.
+<3> Name of the interface.
+<4> The type of interface. This example creates a VRF instance.
+<5> The node interface to which the VRF attaches.
+<6> Name of the route table ID for the VRF.
+<7> Defines the configuration for network routes. `next-hop-address` defines the IP address of the next hop for the route. `next-hop-interface` defines the outgoing interface for the route. In this example, the VRF's routing table is `2`, which references the ID that you define in the `EgressService` CR.
+<8> Defines additional route rules. The `ip-to`` fields must match the cluster IP address and pods CIDR in the networks.operator.openshift.io CR named cluster.
+//Does `oc describe network.config/cluster` get this information?
```

Comment:
@fedepaol - Callout 8 is a bit unclear. Could we make it clearer by getting the exact field names required for the cluster IP address and the pods CIDR. I tried using `oc describe network.config/cluster`, for example but didn't see exactly a field called cluster, but did see fields like `serviceNetwork` etc.

Reply:
So, under cluster you have two values:

```
    clusterNetwork:
    - cidr: 10.132.0.0/14  
```
This ^^^ is the pods CIDR

```
    serviceNetwork:
    - 172.30.0.0/16
```
^^^
This is the clusterip CIDR